### ISSUE TITLE
Translate the UI into Chinese

### ISSUE THREAD

**Issue by programarivm** [chesslablab/website#451]
With the current set up, it is easy-peasy to translate the UI into Russian. It is all about creating a new file in the assets/js/translations folder following the same logic as the [assets/js/translations/messages.es.js](https://github.com/chesslablab/website/blob/main/assets/js/translations/messages.es.js) file which serves as an example to solve this issue.

**Comment by programarivm** [comment]
Thank you @orestonce for contributing to the website repo, please let me know if you'd want to be assigned this one.

**Comment by orestonce** [comment]
Yes. I do it now: https://github.com/chesslablab/website/pull/452
